hello_app/views.py

```diff
@@ -0,0 +1,28 @@
+from datetime import datetime
+from flask import Flask, render_template
+from hello_app import app
+
+@app.route('/')
+def home():
+    return render_template("home.html", title="Home")
```

Comment:
I'd move that to the template using another `{% block %}`

Reply:
I don't understand what you mean. Is there a default rendering for / that we'd take advantage of here?

Reply:
I just meant the `title="..."` since that's something you'd usually put in your template.

The `render_template` calls you need to keep

Reply:
Ah, I see. Thanks.